cmd/kube-proxy/app/server.go

```diff
@@ -171,17 +185,21 @@ func (s *ProxyServer) Run(_ []string) error {
 	dbus := utildbus.New()
 	ipt := utiliptables.New(execer, dbus, protocol)
 
-	shouldUseIptables := false
-	if !s.ForceUserspaceProxy {
+	if !checkKnownProxyMode(s.ProxyMode) {
```

Comment:
Does it matter?  We don't have a clear "check preconditions" block so I put it near where it is used.